solr/solr-ref-guide/src/learning-to-rank.adoc

```diff
@@ -247,6 +254,81 @@ The output XML will include feature values as a comma-separated list, resembling
   }}
 ----
 
+=== Running a Rerank Query Interleaving Two Models
+
+To rerank the results of a query, interleaving two models (myModelA, myModelB) add the `rq` parameter to your search, passing two models in input, for example:
+
+[source,text]
+http://localhost:8983/solr/techproducts/query?q=test&rq={!ltr model=myModelA model=myModelB reRankDocs=100}&fl=id,score
+
+To obtain the model that interleaving picked for a search result, computed during reranking, add `[interleaving]` to the `fl` parameter, for example:
```

Comment:
question: if myModelA had `[ doc1, doc2, doc3 ]` document order and myModelB had `[ doc1, doc3, doc2 ]` document order i.e. there was agreement between the models re: the first document, will `[interleaving]` return (1) randomly `myModelA` or `myModelB` depending on how the picking actually happened or will it return (2) something else e.g. `myModelA,myModelB` (if myModelA actually picked and myModelB agreed) or `myModelB,myModelA` (if myModelB actually picked and myModelA agreed) or will it return (3) neither since in a way neither of them picked the document since they both agreed on it?

answer-ish: from recalling the implementation the answer is (1) i think though from a user's perspective perhaps it might be nice here to clarify here somehow around that? a subtle aspect being (if i understand things right) that `[features]` and `[interleaving]` could both be requested in the `fl` and whilst myModelA and myModelB might have agreed that `doc1` should be the first document they might have used very different features to arrived at that conclusion and their `score` value could also differ.

Reply:
1 is what is currently implemented and it aligns with the TeamDraft Interleaving papers and evaluation methods.
Your observation is interesting though, but to implement that we should invent a new type of Interleaving algorithm that will do that when interleaving the results and will evaluate the user clicks accordingly later on.

Your observation on the features to log applies as well.
So far  no change is needed in this regard in my opinion.